src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRow1099Processor.java

```diff
@@ -925,7 +927,13 @@ private void checkForGeneralExclusionsAndFindTaxBox(ResultSet rs, Transaction109
         int idx;
         
         // Find tax box.
-        if (getPaymentReason1099BoxService().isPaymentReasonMappedTo1099Box(paymentReasonCodeP.value)) {
+        if (getDocumentType1099BoxService().isDocumentTypeMappedTo1099Box(docTypeP.value)) {

```

Comment:
Not sure why there are so many instances of it, but shouldn't it be
```java
docTypeP.getValue()
```
rather than
```java
docTypeP.value
```
?